src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -20,10 +20,6 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     internal sealed class SourceOrdinaryMethodSymbol : SourceOrdinaryMethodSymbolBase
     {
         private readonly TypeSymbol _explicitInterfaceType;
-        private readonly bool _isExpressionBodied;
-        private readonly bool _hasAnyBody;
-        private readonly RefKind _refKind;
-        private bool _lazyIsVararg;
```

Comment:
the big savings come from here.  This is 32bits across all methods (which is a lot for something like roslyn).  This is also because traces show that some of the work we do realizes *all* members in a symbol sometimes (at least if i'm reading the traces right).   So if you have a type with tons of methods, it can be a lot of savings.